Client/Frontend/Toolbar+URLBar/TabLocationView.swift

```diff
@@ -135,6 +147,9 @@ class TabLocationView: UIView, FeatureFlaggable {
                 name: .TabLocationReaderModeAddToReadingListAccessibilityLabel,
                 target: self,
                 selector: #selector(self.readerModeCustomAction))]
+        readerModeButton.showsLargeContentViewer = true
+        readerModeButton.largeContentTitle = .TabLocationReaderModeAccessibilityLabel
+        readerModeButton.largeContentImage = .templateImageNamed("reader")
```

Comment:
I reused an existing accessibility label. Should I create a separate localized string for the `largeContentTitle`?